Content.Shared/Shipyard/SharedShipyardSystem.cs

```diff
@@ -17,7 +17,8 @@ public enum ShipyardConsoleUiKey : byte
     Expedition,
     Scrap,
     // Do not add any ship to this key. Shipyards using it are inherently empty and are populated using the ShipyardListingComponent.
-    Custom
+    Custom,
+    CustomSecurity
```

Comment:
This is no longer Custom, you made it static, no point in calling it custom when its not

Reply:
Isnt it still custom in the sense you have to define the list of ships per console?